.wokeignore

```diff
@@ -0,0 +1 @@
+**
```

Comment:
We can but not really, since the word "blackbox" doesn't pass the check (and as you can imagine, that's pretty much everywhere).

It would be useful if we could `.wokeignore` words instead of files, but this is where we're at!